File: src/main/java/com/haozileung/web/service/permission/IOperationService.java
package com.haozileung.web.service.permission;

import com.haozileung.infra.pager.PageResult;
import com.haozileung.web.domain.permission.Operation;


public interface IOperationService {

    void save(Operation dictionary);

    void update(Operation dictionary);

    void delete(Operation dictionary);

    PageResult<Operation> query(Operation dictionary, int pageNo, int pageSize);
}

File: src/main/kotlin/com/haozileung/web/service/permission/IOperationService.kt
package com.haozileung.web.service.permission

import com.haozileung.infra.pager.PageResult
import com.haozileung.web.domain.permission.Operation


interface IOperationService {

    fun save(dictionary: Operation)

    fun update(dictionary: Operation)

    fun delete(dictionary: Operation)

    fun query(dictionary: Operation, pageNo: Int, pageSize: Int): PageResult<Operation>
}
